swaglyrics/__main__.py

```diff
@@ -10,63 +10,84 @@
 
 
 def main():
-	# 	print(r"""
-	#  ____                     _               _
-	# / ___|_      ____ _  __ _| |   _   _ _ __(_) ___ ___
-	# \___ \ \ /\ / / _` |/ _` | |  | | | | '__| |/ __/ __|
-	#  ___) \ V  V / (_| | (_| | |__| |_| | |  | | (__\__ \
-	# |____/ \_/\_/ \__,_|\__, |_____\__, |_|  |_|\___|___/
-	#                     |___/      |___/
-	# 	""")
-	print('Updating unsupported.txt from server.')
-	with open('unsupported.txt', 'w', encoding='utf-8') as f:
-		response = requests.get('http://aadibajpai.pythonanywhere.com/master_unsupported')
-		f.write(response.text)
-	print("Updated unsupported.txt successfully.")
+    # 	print(r"""
+    #  ____                     _               _
+    # / ___|_      ____ _  __ _| |   _   _ _ __(_) ___ ___
+    # \___ \ \ /\ / / _` |/ _` | |  | | | | '__| |/ __/ __|
+    #  ___) \ V  V / (_| | (_| | |__| |_| | |  | | (__\__ \
+    # |____/ \_/\_/ \__,_|\__, |_____\__, |_|  |_|\___|___/
+    #                     |___/      |___/
+    # 	""")
+    print('Updating unsupported.txt from server.')
+    with open('unsupported.txt', 'w', encoding='utf-8') as f:
+        response = requests.get(
+            'http://aadibajpai.pythonanywhere.com/master_unsupported')
+        f.write(response.text)
+    print("Updated unsupported.txt successfully.")
+    program = "Swaglyrics"
+    parser = argparse.ArgumentParser(
+        prog=program,
+        usage='{prog} [options]'.format(prog=program),
+        description="Get lyrics for the currently playing song on Spotify. Either --tab or --cli is required.")
 
-	parser = argparse.ArgumentParser(
-		description="Get lyrics for the currently playing song on Spotify. Either --tab or --cli is required.")
+    """
+    add_mutually_exclusive_group - makes sure either -c or -t is
```

Comment:
does fitting this in one line exceed 120 chars?

Reply:
Okay no, autopep8 got 79 characters in a line, project default was 120, I think this needs a revert back, then it'll be better.